lib/src/ui/watch/tv_variant_selector.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:lichess_mobile/src/model/tv/tv_variant.dart';
+
+class TvVariantSelector extends ConsumerWidget {
+  const TvVariantSelector({super.key});
+
+  void showChoices(BuildContext context) {
```

Comment:
there's already a `showChoicePicker` function available from `lib/src/widgets/adaptive_choice_picker.dart`; it's a generic function that takes any `Enum`; it would be better to use it to make the app more consistent.